runtime-config-linux.md

```diff
@@ -319,11 +319,44 @@ For more information about Apparmor, see [Apparmor documentation](https://wiki.u
 Seccomp provides application sandboxing mechanism in the Linux kernel.
 Seccomp configuration allows one to configure actions to take for matched syscalls and furthermore also allows matching on values passed as arguments to syscalls.
 For more information about Seccomp, see [Seccomp kernel documentation](https://www.kernel.org/doc/Documentation/prctl/seccomp_filter.txt)
-The actions and operators are strings that match the definitions in seccomp.h from [libseccomp](https://github.com/seccomp/libseccomp) and are translated to corresponding values.
+The actions, architectures, and operators are strings that match the definitions in seccomp.h from [libseccomp](https://github.com/seccomp/libseccomp) and are translated to corresponding values.
+A valid list of constants as of Libseccomp v2.2.3 is contained below.
```

Comment:
On Wed, Sep 23, 2015 at 01:34:39PM -0700, Michael Crosby wrote:
> @mheon maybe we should not use the actual values that libseccomp
> expects because it would be hard to version and have spec stability
> if some low level detail like what strings libseccomp expects leaks
> out via the spec.

The bundle author will need to know what to use, regardless of whether
we list these in the spec ;).  So one way is to code the allowed
values into the spec, so it's a contract between the spec and bundle
author.  Another way is to allow contract directly between the
runtime's libseccomp and the bundle author (so the spec just provides
a way for the author to specify the libseccomp version they're writing
to).  Maybe this is a use-case for my proposed version structure [1]?
The bundle author would use something like:

  "version": {
    "openContainerSpecification": "0.2.0",
    "libseccomp": "2.2.0",
  }

And then be compatible with runtimes that support spec versions <0.3.0
which are compiled against libseccomp versions <3.0.0 (assuming
libseccomp semantically versions their header).

[1]: https://github.com/opencontainers/specs/issues/110#issuecomment-133587646